index.js

```diff
@@ -96,20 +163,22 @@ walletRouter.post('/simple-send-tx', (req, res) => {
   });
 });
 
+walletRouter.post('/stop', (req, res) => {
+  // Stop wallet and remove from wallets object
+  const wallet = req.wallet;
+  wallet.stop();
+
+  const key = req.body.key;
+  delete wallets[key];
+  res.send({success: true});
+});
+
 if (config.http_api_key) {
   app.use(apiKeyAuth(config.http_api_key));
 }
-app.use(express.json());
-app.use(express.urlencoded({ extended: true }));
-app.use('/0', walletRouter);
+app.use('/wallet', walletRouter);
 
 app.listen(config.http_port, config.http_bind_address, () => {
   console.log(`Hathor Wallet listening on ${config.http_bind_address}:${config.http_port}...`);
-  wallet.start().then((info) => {
```

Comment:
Maybe we should have a parameter in the config (or in the args from the cmdline) to auto start a wallet.

Reply:
The user would have to select which `seedKey` and which `key` he wants to auto start. Even though I think it's good to have it, I think it's more important when you want a single wallet.

In my opinion we should discuss adding an easier support for single wallets in another PR.